pages/_document.js

```diff
@@ -22,6 +22,11 @@ export default class MyDocument extends Document {
         <Head>
           <title>Register a food business</title>
           <link rel="stylesheet" href="/_next/static/style.css" />
+          {/* TODO: remove Libre Franklin, replace with official GOV.UK font */}
```

Comment:
Has this TODO been tracked anywhere? How long will this take / what is blocking it?

Reply:
Where should I be tracking it? The blocker is again to be discussed tomorrow, it's to do with the actual gov font not being easily accessible or licensed for any websites that aren't on gov.uk